### ISSUE TITLE
generated .wav files from pcm are not playable in OS X Quicklook

### ISSUE THREAD

**Issue by WeeJeWel** [TooTallNate/node-wav#15]
Maybe the file (headers?) are corrupt. Another .wav can be played in Quicklook (press spacebar when the file is selected).

Attached a .zip with the .pcm and the generated .wav (using https://github.com/athombv/pcm2wav)

[what-is-the-weather.zip](https://github.com/TooTallNate/node-wav/files/136213/what-is-the-weather.zip)

**Comment by LinusU** [comment]
Yep, this is because the FileWriter is broken, #17 fixes this a bit (but I think it will cut the first 44 bytes which will probably misalign the rest of the file)...
